dashboard/test/ui/features/applab/eyes.feature

```diff
@@ -158,3 +158,17 @@ Scenario: Applab Instructions Resize
   Then I am on "http://learn.code.org/s/allthethings/stage/18/puzzle/9?topInstructions=false"
   And I wait to see "#runButton"
   Then I close my eyes
+
+Scenario: Applab debugging
+  When I open my eyes to test "Applab debugging"
+  And I am on "http://learn.code.org/projects/applab/new"
```

Comment:
It might be more robust and concise to use `And I start a new Applab project` since it does a few additional checks